src/main/java/net/minecraftforge/gui/ForgeGuiHandler.java

```diff
@@ -0,0 +1,51 @@
+package net.minecraftforge.gui;
+
+import io.netty.channel.embedded.EmbeddedChannel;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.inventory.Container;
+import net.minecraft.world.World;
+import net.minecraftforge.common.util.FakePlayer;
+import net.minecraftforge.fml.common.FMLCommonHandler;
+import net.minecraftforge.fml.common.network.FMLOutboundHandler;
+import net.minecraftforge.fml.common.network.internal.FMLNetworkHandler;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.gui.capability.IGuiProvider;
+
+public class ForgeGuiHandler
+{
+    public static void openGui(EntityPlayer player, World worldObj, IGuiProvider provider)
+    {
+        if(provider == null) return;
+        Object owner = provider.getOwner(player, worldObj);
+        if(FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT)
```

Comment:
if you're going to use getEffectiveSide then might as well check world.isRemote. Same semantics, but second is more reliable